solarkraft/test/e2e/tla/.stor/setter/1741597/entry-df673ab5399dcbd5ac434c0ac51c53e9b9367b1d2507ab7e4b8cebfe3e796499.json

```diff
@@ -0,0 +1 @@
+{"height":1741597,"txHash":"df673ab5399dcbd5ac434c0ac51c53e9b9367b1d2507ab7e4b8cebfe3e796499","contractId":"CBOAR2NNJP77RBSZ7XHTGZGX3BRYGLDUNFRD2CPL5FRKIBBU77XYJCMD","method":"set_bytes","methodArgs":[{"type":"Buffer","data":[190,239]}],"returnValue":null,"fields":[["MY_ADDR","CCLCX5GS3NRUHDLKFBTHWT5CD5US5BUNWCX6H2BN3RKXMVUZMYRXJLBT"],["MY_BOOL",true],["MY_BTES32",{"type":"Buffer","data":[253,237,63,85,222,196,114,80,165,42,140,11,183,3,142,114,250,111,250,174,51,86,47,119,205,43,98,158,247,253,66,77]}],["MY_BYTES",{"type":"Buffer","data":[190,239]}],["MY_ENUM",["A",0]],["MY_I128",-42],["MY_I32",-42],["MY_I64",-42],["MY_MAP",{}],["MY_STRUCT",{"a":0,"b":0}],["MY_SYM","hello"],["MY_U128",42],["MY_U32",42],["MY_U64",42],["MY_VEC",[]]],"oldFields":[["MY_ADDR","CCLCX5GS3NRUHDLKFBTHWT5CD5US5BUNWCX6H2BN3RKXMVUZMYRXJLBT"],["MY_BOOL",true],["MY_BTES32",{"type":"Buffer","data":[253,237,63,85,222,196,114,80,165,42,140,11,183,3,142,114,250,111,250,174,51,86,47,119,205,43,98,158,247,253,66,77]}],["MY_BYTES",{"type":"Buffer","data":[0]}],["MY_ENUM",["A",0]],["MY_I128",-42],["MY_I32",-42],["MY_I64",-42],["MY_MAP",{}],["MY_STRUCT",{"a":0,"b":0}],["MY_SYM","hello"],["MY_U128",42],["MY_U32",42],["MY_U64",42],["MY_VEC",[]]]}
```

Comment:
there is a way to add these files to a github ignore list, so they are collapsed by default

Reply:
Oh true, that's amazing!
https://github.com/github-linguist/linguist/blob/master/docs/overrides.md#generated-code